pkgs/apps/wxmacmolplt/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, requireFile, wxGTK, libGL, libGLU, pkg-config, xorg, breakpointHook }:
+
+stdenv.mkDerivation rec {
+  pname = "wxmacmolplt";
+  version = "7.7.2";
+
+  /* There is a redirection to another url and it is not directly a file.
+  src = fetchurl {
+    url = "https://uofi.box.com/shared/static/ezm1jd56hlb8s2vsf9ou28vrctjqqa9o.gz";
+    hash = "sha256-7vI8J8ZkYWx0D4gFph2378BV7Y211puvzsaer6/il+Q=";
+  };
+  */
+
+  src = requireFile {
```

Comment:
The sources are hosted at Github: https://github.com/brettbode/wxmacmolplt
Why not just fetch either the release tag from there or download the provides source files from there?

Reply:
I am afraid because I've just overlooked it. Uses Github now. Thanks for the hint :slightly_smiling_face: